.drone.yml

```diff
@@ -464,18 +473,18 @@ steps:
       cron: security_scans
       event: cron
 
-  - name: cron_anchore_scan
-    image: 340268328991.dkr.ecr.eu-west-2.amazonaws.com/acp/anchore-submission:latest
+
+  - name: cron_trivy_scan
+    image: 340268328991.dkr.ecr.eu-west-2.amazonaws.com/acp/trivy/client:latest
     pull: always
     environment:
-      IMAGE_NAME: ima:${DRONE_COMMIT_SHA}
-      LOCAL_IMAGE: true
-      TOLERATE: medium
-      WHITELIST_FILE: hof-services-config/Illegal_Migration_Act/anchore-cve-exceptions.txt
```

Comment:
should we have a WHITELIST_FILE for trivy as well?

Reply:
We have an option to add similar to WHITELIST_FILE  for Trivy which is "ALLOW_CVE_LIST_FILE".. @JHoldergov  Please Advice

Reply:
Can anyone of you help me locate the Whitelist file please. I will create one for Trivy and add the file path to pipeline.